lib/movie.dart

```diff
@@ -0,0 +1,53 @@
+class Movie {
+  final String title;
+  final String originalTitle;
+  final String overview;
+  final DateTime releaseDate;
+  final double voteAverage;
+  final List<String> genres;
+  final String poster;
+  final String backdrop;
+  int likes = 0;
+
+  Movie(
+    this.title,
+    this.originalTitle,
+    this.overview,
+    this.releaseDate,
+    this.voteAverage,
+    this.genres,
+    this.poster,
+    this.backdrop,
+  );
+
+  void likeMovie() => likes++;
+
+  Movie.marioBrosConstructor(
+      {this.title = "The Super Mario Bros. Movie",
+      this.originalTitle = "The Super Mario Bros. Movie",
+      this.overview = "While working underground to fix a "
+          "water main, Brooklyn plumbers—and brothers—Mario and Luigi are "
+          "transported down a mysterious pipe and wander into a magical new "
+          "world. But when the brothers are separated, Mario embarks on an epic "
+          "quest to find Luigi.",
+      releaseDate,
+      this.voteAverage = 7.8,
+      this.genres = const [
+        "Animation",
+        "Family",
+        "Adventure",
+        "Fantasy",
+        "Comedy"
+      ],
+      this.poster = 'movie_images/poster.jpg',
+      this.backdrop = 'movie_images/backdrop.jpg'})
+      : releaseDate = DateTime(2023, 4, 5);
```

Comment:
You can use a factory constructor, and also think that the name could be changed it, what happened if tomorrow i change the mario movie template with another movie like Harry Potter? This method name make sense to you with that change?

Reply:
fixed